tests/chainer_tests/functions_tests/normalization_tests/test_batch_normalization.py

```diff
@@ -210,6 +215,15 @@ def check_forward(self, inputs, backend_config):
                 **self.check_forward_options)
 
     def test_forward(self, backend_config):
+        if isinstance(backend_config.device, chainer.backend.Intel64Device):
+            # ideep only supports float32
```

Comment:
Please skip the test. `raise unittest.SkipTest('ideep only supports float32')`
Ditto for other tests.